tests/api/lib/indexing_pipeline_test.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+
+from haystack import Pipeline, Document
```

Comment:
On executing `pytest`, warning generated:

```
=================================================== warnings summary ===================================================
search-backend-env/lib/python3.10/site-packages/haystack/core/errors.py:34
  /Users/richard.leyshon/Documents/GitHub/MoJ/rd-search-backend/search-backend-env/lib/python3.10/site-packages/haystack/core/errors.py:34: DeprecationWarning: PipelineMaxLoops is deprecated and will be remove in version '2.7.0'; use PipelineMaxComponentRuns instead.
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```
haystack warning - [see here](https://github.com/deepset-ai/haystack/issues/8452 ),  incrementing to haystack 2.7 should get rid of this when it's released (still on 2.6.1 for now)

Reply:
If @chloepugh thinks we can upgrade, we probably should.